nixos/modules/services/desktops/pipewire/pipewire.nix

```diff
@@ -200,6 +224,15 @@ in {
           '';
         };
       };
+
+      configPackages = lib.mkOption {
+        type = lib.types.listOf lib.types.package;
+        default = [];
+        description = lib.mdDoc ''
+          List of packages that provide pipewire configuration, in the form of
+          `share/pipewire/*/*.properties` files.
```

Comment:
Nit: it's not actually `.properties` I think? Unless Pipewire also accepts that? But it's usually .conf or .json